src/pages/Social.jsx

```diff
@@ -4,82 +4,107 @@ import BlueDiscord from '../assets/blueDiscord.png';
-                    fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt 
-                    mollit anim id est laborum.
-                </p>
-            </div>
-            <div className="flex flex-row mt-40 gap-5">
-                <img src={FusionFriday} alt={`Fusion Friday`} className="w-307 h-307 object-cover flex-shrink-0 mt-10"/>
-                <img src={FusionFriday} alt={`Fusion Friday`} className="w-307 h-307 object-cover flex-shrink-0 mb-10"/>
-                <img src={FusionFriday} alt={`Fusion Friday`} className="w-307 h-307 object-cover flex-shrink-0 mt-10"/>
-                <img src={FusionFriday} alt={`Fusion Friday`} className="w-307 h-307 object-cover flex-shrink-0 mb-10"/>
-                <img src={FusionFriday} alt={`Fusion Friday`} className="w-307 h-307 object-cover flex-shrink-0 mt-10"/>
-            </div>
-            <div className="max-w-[1088px] w-full flex flex-col justify-center items-center gap-10 mt-60">
-                <h1 className="text-black text-center font-inter text-4xl font-medium" style={{ letterSpacing: '8px' }}>
-                    FTEK
-                </h1>
-                <p className="text-black text-center font-inter text-base leading-7 px-36">
-                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et 
-                    dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip 
-                    ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu 
-                    fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt 
-                    mollit anim id est laborum.
-                </p>         
-            </div>
-            <div className="flex flex-row mt-40 gap-20 overflow-hidden justify-center">
-                <img src={Ftek} alt={`Ftek`} className="w-857 h-510 object-cover flex-shrink-0 mt-10"/>
-                <img src={Ftek} alt={`Ftek`} className="w-857 h-510 object-cover flex-shrink-0 mt-10"/>
-                <img src={Ftek} alt={`Ftek`} className="w-857 h-510 object-cover flex-shrink-0 mt-10"/>
-            </div>
-            <div className="max-w-[1088px] w-full flex flex-col justify-center items-center gap-10 mt-60">
-                <h1 className="text-black text-center font-inter text-4xl font-medium" style={{ letterSpacing: '8px' }}>
-                    GAMING
-                </h1>
+
+            <div className="w-full bg-[#F4F4F4] flex mt-40 overflow-x-hidden">
+                <div className="max-w-[1088px] w-full flex flex-col gap-10 relative">
+                    <img src={WhiteHalfSun} alt={`White Half Sun`} className="w-1211.6 h-964.12 left-0 object-cover flex-shrink-0 z-0"/>
```

Comment:
when working with these width and height values, I would suggesst not exceeding 96 as that's what's listed in the tailwnid documentation and im not sure if it would work properly like that with the 1211. Check out the width page and it offers more about using w-1/2 and stuff like that